service/src/main/java/uk/gov/hmcts/reform/fpl/utils/AgeDisplayFormatHelper.java

```diff
@@ -13,27 +12,35 @@ private AgeDisplayFormatHelper() {
 
     public static String formatAgeDisplay(final LocalDate dateOfBirth) {
         checkNotNull(dateOfBirth, "Date of birth value is required");
-        checkArgument(!dateOfBirth.isAfter(LocalDate.now()), "Date of birth cannot be in future");
 
-        Period period = Period.between(dateOfBirth, LocalDate.now());
-        int years = period.getYears();
-        int months = period.getMonths();
-        int days = period.getDays();
+        final Period period = Period.between(dateOfBirth, LocalDate.now());
+        final int years = period.getYears();
+        final int months = period.getMonths();
+        final int days = period.getDays();
 
+        if (period.isNegative()) {
+            return "0 years old";
```

Comment:
Does the last return not cover this?

Reply:
last return says '0 days old' not '0 years old' and is used when child born today

Reply:
Could it not be updated to just return the same? Or do we want the explicit difference?

Reply:
this is what was agreed with Sara today on standup.

Reply:
sure thing